src/WeekComponent/WeekComponent.js

```diff
@@ -19,84 +20,92 @@ class WeekComponent extends React.Component {
     this.state = {
       seconds: 0,
       totalSeconds: this.total(exercises[this.weekNr][this.dayNr].machine),
-      action: "idlee"
+      action: "idle"
     }
   }
 
-  total(machine){
+  writeToLocalStorage = () => {
+    localStorage.setItem('weekNumber', this.weekNr);
+    localStorage.setItem('dayNumber', this.dayNr)
+  };
+
+  total(machine) {
     let totali = 0;
-    for(let i = 0; i < machine.length; i++){
+    for (let i = 0; i < machine.length; i++) {
       totali += machine[i];
     }
-    return totali; 
+    return totali;
   }
 
   componentWillUnmount() {
-    if(this.state.intervalId) {
+    if (this.state.intervalId) {
       clearInterval(this.state.intervalId);
     }
   }
 
   start() {
     this.setState({
-      action:this.oldAction?this.oldAction:"warmup"
+      action: this.oldAction ? this.oldAction : "warmup"
     }, this.stopwatch);
   }
 
-  pause(){
-    if(this.state.intervalId) {
+  pause() {
+    if (this.state.intervalId) {
       clearInterval(this.state.intervalId);
       this.oldAction = this.state.action;
       this.setState({
-        action:"paused",
+        action: "paused",
         intervalId: null
       });
     }
   }
 
-  toMins(seconds){
-    return Math.floor(seconds/60) + ":" + (seconds % 60 < 10 ? "0" + seconds % 60 : seconds % 60);
+  toMins(seconds) {
```

Comment:
@KaltrinaDragobuzhda can we rename this to `convertSecondsToMinutes` or something similar, because right now we can't tell what is being converted to minutes without checking the argument's name